### PR TITLE
fix: display details for legacy multi signature transactions

### LINKED ISSUE #852: bug: no info for legacy multisignature registration transactions
No information is displayed for legacy multisignature registration transactions aside from the transaction id.

**Expected Behavior**

Information about the legacy multisignature registration should be shown (at least things like sender, fee, confirmations, block id).

**Current Behavior**

Only the transaction ID is shown.
 
**Steps to Reproduce (for bugs)**

See https://dexplorer.ark.io/transaction/32aa60577531c190e6a29d28f434367c84c2f0a62eceba5c5483a3983639d51a

### PR DESCRIPTION
**Pull request by dated**
## Summary

Fixes #852.

## Checklist

- [ ] Documentation _(if necessary)_
- [x] Tests _(if necessary)_
- [x] Ready to be merged

### CODE REVIEW

File: src/components/transaction/Details.vue [hunk 1]
```diff
@@ -155,13 +155,13 @@
...
 
-    <section v-if="isMultiSignature(transaction.type, transaction.typeGroup)" class="page-section py-5 md:py-10 mb-5">
+    <section v-if="isMultiSignature(transaction.type, transaction.typeGroup)" class="TransactionDetails__MultiSignature page-section py-5 md:py-10 mb-5">
       <div class="px-5 sm:px-10">
         <div class="list-row-border-b">
           <div class="mr-4">{{ $t("TRANSACTION.MULTI_SIGNATURE.ADDRESS") }}</div>
           <div class="truncate">
             <LinkWallet
-              :address="addressFromMultiSignatureAsset(transaction.asset.multiSignature)"
+              :address="addressFromMultiSignatureAsset(multiSignatureAsset)"
```
Comment (ItsANameToo) [inline src/components/transaction/Details.vue:164]: the legacy multisig wallets did not generate a new wallet. Instead they made the sending wallet a multisignature one 🤔 So this now shows a wallet address for legacy multisig that does not exist

**Review by ItsANameToo — CHANGES_REQUESTED** [review]

File: src/components/transaction/Details.vue (hunk 1 above)
Comment (dated) [reply src/components/transaction/Details.vue:164]: Oh, I did not know that. I'll adjust it asap.

Comment (dated) [reply src/components/transaction/Details.vue:164]: Should be good to go now, see 713eb1a and 5e6e89d.

Comment (dated) [reply src/components/transaction/Details.vue:164]: The "Generated address" line will now show only for non-legacy transactions.

**Review by ItsANameToo — APPROVED** [review]

**Comment by ghost** [comment]
A member has approved this PR. A maintainer will merge this PR shortly. If it shouldn't be merged yet, please leave a comment saying so and we'll wait.

_Thank you for your contribution!_

**Comment by ghost** [comment]
Your pull request has been merged and marked as tier 4. _It will earn you $20 USD._

**Pull request merged**